front/SettingsForm.tsx

```diff
@@ -107,6 +111,7 @@ const SettingsForm = () => {
           <button
             className="btn btn-lg btn-primary pull-xs-right"
             type="submit"
+            data-cy = 'update-btn'
```

Comment:
Remove the space between 'data-cy' and '='. It should be written as 'data-cy="update-btn"'